api/SVGStringList.json

```diff
@@ -77,10 +77,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": "13.1"
```

Comment:
This is `length`, I don't trust this data. Needs to test an instance to be sure.